ossindex/ossindex.go

```diff
@@ -78,7 +78,9 @@ func AuditPackages(purls []string) ([]types.Coordinate, error) {
 
 	defer func() {
 		if err := db.Close(); err != nil {
-			log.Printf("error closing db: %s\n", err)
+			if !quiet {
```

Comment:
Rather than doing a lot of `if !quiet` statements, what if we had a helper method that takes the `log.Printf` or whatever statement, and does the check in there? Then you could not have to pass `quiet` all over the place too?

Reply:
@DarthHater Ok if checks removed in the latest commit. Realized that `log` has the ability to just set the output. So if i just set the output to be `Discard` it would get us the same result.